### PR TITLE
Updated translations

### PR DESCRIPTION
**Pull request by ThexXTURBOXx**
- Added Discord link to intros
 - Updated German translations

### CODE REVIEW

File: src/main/resources/translations/html/intro.english.html [hunk 1]
```diff
@@ -131,6 +131,7 @@ <h2>Notes</h2>
     <li>If BCV fails to boot simply append -clean as an argument to clean the lib directory.</li>
     <li>Relax and take notes, while I take tokes of the marijuana smoke.</li>
     <li>BCV was created out of love for Java Reverse engineering.</li>
+	<li>You can join our Discord server at <a href="https://discord.gg/aexsYpfMEf">https://discord.gg/aexsYpfMEf</a>!</li>
```
Comment (Janmm14) [inline src/main/resources/translations/html/intro.english.html:134]:
```suggestion
    <li>You can join our Discord server at <a href="https://discord.gg/aexsYpfMEf">https://discord.gg/aexsYpfMEf</a>!</li>
```

File: src/main/resources/translations/html/intro.mandarin.html [hunk 2]
```diff
@@ -130,6 +130,7 @@ <h2>注意事项</h2>
         <li>如果 BCV 无法启动，只需添加 -clean 作为参数来清理 lib 目录。
         <li>在我吸食大麻烟的时候，请放松并做笔记。
         <li>BCV 是出于对 Java 反向工程的热爱而创建的。
+		<li>你可以在<a href="https://discord.gg/aexsYpfMEf">https://discord.gg/aexsYpfMEf</a>加入Discord服务器!</li>
```
Comment (Janmm14) [inline src/main/resources/translations/html/intro.mandarin.html:133]:
```suggestion
        <li>你可以在<a href="https://discord.gg/aexsYpfMEf">https://discord.gg/aexsYpfMEf</a>加入Discord服务器!</li>
```

**Comment by ThexXTURBOXx** [comment]
Thanks @Janmm14

**Comment by Konloch** [comment]
Thank you!

**Pull request merged**